code/game/objects/items/storage/pouch.dm

```diff
@@ -118,23 +118,24 @@
 	name = "first-aid pouch"
 	desc = "Standard marine first-aid pouch. It can contain autoinjectors, sets of pills, and bandages."
 	icon_state = "firstaid"
-	storage_slots = 6
+	storage_slots = 7
 	can_hold = list(
```

Comment:
No

Reply:
Medkit pouch is the same amount of slots, this doesn't really change anything in the long run.